lock-provider/src/test/java/com/inomera/telco/commons/lock/zookeeper/ZookeeperLockProviderTest.java

```diff
@@ -0,0 +1,344 @@
+package com.inomera.telco.commons.lock.zookeeper;
+
+import com.inomera.telco.commons.lock.Locked;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.retry.ExponentialBackoffRetry;
+import org.apache.curator.test.TestingServer;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Supplier;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ZookeeperLockProviderTest {
+
+    private static final String DEFAULT_PATH = "/tmp/locks";
+
+    private ZookeeperLockProvider lockProvider;
+
+    TestingServer zkServer;
+    CuratorFramework curatorClient;
```

Comment:
zkServer and curatorClient fields can be defined as `private`.

```suggestion
    private TestingServer zkServer;
    private CuratorFramework curatorClient;
```